lib/engines/content_block_manager/app/models/content_block_manager/content_block/document/scopes/searchable_by_keyword.rb

```diff
@@ -2,6 +2,12 @@ module ContentBlockManager
   module ContentBlock::Document::Scopes::SearchableByKeyword
     extend ActiveSupport::Concern
 
+    SQL = <<-SQL.freeze
```

Comment:
This is much neater 👍

Reply:
Now I've remembered Arel exists, we might be able to use that to improve this - not for now though :)

Reply:
yes I did have a quick look at Arel to see if it could be tidied up further but tbh I just got more confused 🙈